packages/frontend/src/utils/ledger.js

```diff
@@ -23,4 +23,8 @@ async function createLedgerU2FClient() {
     return client;
 }
 
-export { createLedgerU2FClient };
+function getLedgerHDPath(confirmedPath){
```

Comment:
Great call for extracting it as a separate function. Just a nit with code formatting/indentation.